src/main/java/liquibase/ext/hibernate/snapshot/TableSnapshotGenerator.java

```diff
@@ -45,7 +45,7 @@ protected DatabaseObject snapshotObject(DatabaseObject example, DatabaseSnapshot
         Scope.getCurrentScope().getLog(getClass()).info("Found table " + table.getName());
 //        table.setSnapshotId(SnapshotIdService.getInstance().generateId());
         table.setSchema(example.getSchema());
-
+        table.setRemarks(hibernateTable.getComment());
```

Comment:
If comment is null, won't this throw a NPE?

Reply:
No. Because `setRemarks` allow `null`